hack/verify-shellcheck.sh

```diff
@@ -64,7 +64,7 @@ fi
 
 echo "Running shellcheck..."
 cd "${ROOT_PATH}" || exit
-IGNORE_FILES=$(find . -name "*.sh" | grep "third_party\|tilt_modules")
+IGNORE_FILES=$(find . -name "*.sh" | { grep "tilt_modules" || true; })
```

Comment:
@oscr thanks for taking care of this
if I'm not wrong there are no more tilt_modules anymore, so we can cleanup this part and remove one of the two -not -path in L69